tests/commands/new/test_validate_app_name.py

```diff
@@ -25,6 +25,9 @@ def test_valid_app_name(new_command, name):
         '_helloworld',  # leading underscore
         '-helloworld',  # leading hyphen
         'existing',  # pre-existing directory
+        'switch',
+        'false',  # added from the keywords_list
```

Comment:
You may want to check some different capitalisations here; `false` and `False` should both be rejected.